app/main.py

```diff
@@ -1,8 +1,37 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        name = person["name"]
+        person_instance = Person.people[name]
+
+        wife_name = person.get("wife")
+        husband_name = person.get("husband")
+
+        if wife_name is not None:
+            wife_instance = Person.people.get(wife_name)
+            if wife_instance:
+                person_instance.wife = wife_instance
+                wife_instance.husband = person_instance
+        else:
+            if hasattr(person_instance, "wife"):
+                del person_instance.wife
```

Comment:
why do you delete it and after that you add it again?
please rewrite code, if person has wife then add wife and no more

Reply:
same with husband